docs/UserGuide/System Tools/NodeTool.md

```diff
@@ -0,0 +1,288 @@
+    under the License.
+
+-->
+
+# Introduction
+IoTDB cluster version provides nodetool shell tool for users to monitor the working status of the specified cluster. 
+Users can obtain the status of the cluster by running a variety of instructions.
+
+The following describes the usage and examples of each instruction, 
+where $IOTDB_CLUSTER_ Home indicates the path of the IoTDB Distributed installation directory.
+# Instructions
+## Get Started
+The nodetool shell tool startup script is located at $IOTDB_CLUSTER_HOME/bin folder, 
+you can specify the IP address and port of the cluster at startup.
+
+IP is the IP of the node that the user expects to connect to,
+and port is the JMX service port specified when the IoTDB cluster is started.
+
+The default values are 127.0.0.1 and 31999, respectively.
+
+If you need to monitor the remote cluster or modify the JMX service port number,
+use the actual IP and port at the -H and -P entries.
+
+## Explains
+In a distributed system, a node is identified by node IP, metadata port and data port \<METAPORT:DATAPORT>.
+### Show The Ring Of Node
+IoTDB cluster version uses consistent hash to achieve data distribution.
+
+Users can know the location of each node in the ring by printing hash ring information.
+
+1.Input
+> ring
+
+2.Output
+
+> The output is a multi line string, and each line of string is a key value pair, 
+> where the key represents the token value and the value represents the node (IP:METAPORT:DATAPORT), the format is \<key -> value>.
+
+3.Examples
+
+> Suppose that the current cluster runs on three nodes: 127.0.0.1:9003:40010, 127.0.0.1:9005:40012 and 127.0.0.1:9004:40011.
+> 
+> Examples of input instructions for different systems are as follows:
+
+Linux and MacOS：
+
+```
+Shell > ./bin/nodetool.sh -h 127.0.0.1 -p 31999 ring
+```
+
+Windows：
+
+```
+Shell > \bin\nodetool.bat -h 127.0.0.1 -p 31999 ring
+```
+
+Press enter to execute the command. 
+
+The output of the example instruction is as follows:
+```
+Node Identifier                                 Node 
+330411070           ->          127.0.0.1:9003:40010 
+330454032           ->          127.0.0.1:9004:40011 
+330496472           ->          127.0.0.1:9005:40012
+```
+ 
+The above output shows that there are three nodes in the current cluster,
+and the output results are output clockwise according to their positions in the ring.
+
+### Query data partition and metadata partition
+The time series metadata of iotdb cluster version is divided into multiple data partitions according to storage groups,
+in which the storage group and data partition are many to one relationship.
+
+That is, the same storage group only exists in the same data partition,
+and a data partition contains multiple storage groups.
+
+The data is divided into multiple data partitions according to the storage group and time interval,
+and the partition granularity is <storage group, time range>.
+
+The data partition is composed of replica nodes,
```

Comment:
I think it's best to state here that a time partition consists of a raft group, thus ensuring high availability of data.